bucket/droidcam.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "v6.3.3",
+    "homepage": "https://dev47apps.com",
+    "description": "Use your phone as a webcam for your PC!",
+    "license": "Freeware",
+    "depends": "main/adb",
+    "shortcuts":    [
+        [
+            "DroidCamApp.exe",
+            "DroidCam"
+        ]
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://files.dev47apps.net/win/DroidCam.Setup.6.3.3.exe#/dl.7z",
+            "hash": "sha512:48f4b5bbe73569df513137a254a16b90a0008f764c9d79aae2ca748fbc54cbf4544e9faa4b0cb5fcef81f693ccde175c696a5fad5fa2176cbc7926297f5c19e1"
+        }
+    },
+    "notes": [
+        "For instructions on connecting, visit https://www.dev47apps.com/droidcam/connect/.",
+        "You may need to install the appropriate USB driver for your Android device to get this to work.",
+        "Please visit https://developer.android.com/studio/run/oem-usb#Drivers for more information."
+    ]
```

Comment:
```suggestion

    "version": "6.3.3",
    "description": "Allows to use phone as a webcam for PC.",
    "homepage": "https://dev47apps.com/",
    "license": "Freeware",
    "notes": "See 'https://www.dev47apps.com/droidcam/help/' for configuration steps.",
    "suggest": {
        "vcredsit": "vcredist"
    },
    "architecture": {
        "64bit": {
            "url": "https://files.dev47apps.net/win/DroidCam.Setup.6.3.3.exe#/dl.7z",
            "hash": "0fb1f9b6526cbf0175d152017a99c050a87746948360c807297075117dd233e8"
        }
    },
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\vc_red*\", \"$dir\\Uninstall*\" -Recurse",
    "shortcuts": [
        [
            "DroidCamApp.exe",
            "DroidCam"
        ]
    ],
    "checkver": {
        "url": "https://www.dev47apps.com/droidcam/windows/",
        "regex": "v([\\d.]+)\\s+Download"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://files.dev47apps.net/win/DroidCam.Setup.$version.exe#/dl.7z"
            }
        }
    }
```

Reply:
DroidCam has a soft dependency on adb for Android devices, its target audience. Without it it will not work.

Reply:
It has it already...

Reply:
Apologies, I didn't see that detail.